haproxy/tests/errors_generate.spec.js

```diff
@@ -0,0 +1,65 @@
+const { expect } = require('chai');
```

Comment:
We typically use a `-`to separate words in file names. I know these existent files did not honor this convention. Would you mind renaming your newly added files with a `-` instead and make the world a little bit more consistent? :D